content/en/docs/architecture/jira.md

```diff
@@ -22,6 +22,22 @@ To rerun the validations, comment `/jira refresh` on the PR. This is mostly usef
 All Openshift bugs must be made in the `OCPBUGS` project. For PRs referencing issues from other projects, a link will be
 created but the rest of the lifecycle management will not be performed for the issue.
 
+## Premerge Verification
+
+The Jira GitHub integration provides a method to mark a PR as `verified` before the PR is merged. This allows the integration
+to move the bug to the `VERIFIED` state upon merge instead of requiring a QE engineer to test and manually move an issue to the
+`VERIFIED` state after the issue moves into the `ON_QA` state. Premerge verfication is strongly preferred.
+
+To mark a PR as `verified`, comment `/verified by reason` on the PR. The reason can be the names of tests, GitHub usernames,
+documentation, or links. Multiple reasons can be listed, separated by a comma. Example: `/verified by @user` or
+`/verified by test1,test2`. If successful, the bot will comment on the PR and add the `verified` label to the PR.
```

Comment:
Is `verified` removed on a new commit?
It occurs to be we may also need `/verified remove`

Reply:
The integration does not remove the verified label on new commits. I can add functionality to do so as well as a `/verified remove`. Since we upload the statistics to BigQuery when the verification comment is made, we'll probably also need to upload remove statistics to BigQuery as well.